packages/react-sdk/src/components/Shortcuts/ClipboardShortcuts/ClipboardShortcuts.tsx

```diff
@@ -218,15 +206,30 @@ export function ClipboardShortcuts() {
         } else {
           const newElements: Elements = {};
           for (const [elementId, element] of Object.entries(elements)) {
-            newElements[elementId] = modifyElementPosition(
+            let newElement = modifyElementPosition(
               element,
               positionClamp,
               offsetX,
               offsetY,
             );
+
+            /**
+             * If pasted element is not attached to frame, then
+             * try to find an existing frame to attach to.
+             */
+            if (
+              newElement.type !== 'frame' &&
+              newElement.attachedFrame === undefined
+            ) {
+              newElement = copyElementWithAttachedFrame(
+                newElement,
+                slideInstance.getFrameElements(),
+              );
+            }
+            newElements[elementId] = newElement;
```

Comment:
As far as I know, this is safe and can't lead to [Prototype Polution](https://stackoverflow.com/a/60850027). 👍

Keep coding as if the next 100 boards I'm going to share with you, will contain the `elementId`s "__proto__" and "constructor". 😈